docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_WALLETDIR: "/data/wallet"
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        rpcauth=eclair:d3a244200d5c946f6bef48e6583ec979$$215049513aef33245dfec8ff3ab5e749458fde51ede931dec04c767c0732d468
+  eclair_bitcoin:
+    container_name: btcpayserver_eclair_bitcoin
+    restart: unless-stopped
+    stop_signal: SIGINT
+    image: acinq/eclair:eclair_cli_in_docker
```

Comment:
I forgot to mention but this should be updated to `acinq/eclair:v0.3.2`, or if @NicolasDorier wants the `/getnewaddress` API i can tag a newer version and can use that one.

Reply:
updated. The `getnewaddress` part is not needed for initial release :)